src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1501,7 +1503,9 @@
 	function m.platformToolset(cfg)
 		local tool, version = p.config.toolset(cfg)
 		if version then
-			version = "v" .. version
+			if tonumber(version:sub(1,3)) ~= nil then
+				version = "v" .. version
```

Comment:
@starkos Are you particularly attached to the functionality of canonical() where it returns `120` instead of `v120`?
Why not just make that function return `v120`, rather than stripping the 'v', and then adding it back on here?